lib/openstax/rescue_from/engine.rb

```diff
@@ -3,6 +3,12 @@
 module OpenStax
   module RescueFrom
     class Engine < ::Rails::Engine
+      initializer 'openstax.rescue_from.inflection' do
+        ActiveSupport::Inflector.inflections do |inflect|
+          inflect.acronym 'OpenStax'
```

Comment:
ah you already have it.... so why `"Openstax"` above?

Reply:
Could not get it to work for the generator...